src/Common/src/System/Data/Common/BasicFieldNameLookup.cs

```diff
@@ -0,0 +1,146 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+
+//------------------------------------------------------------------------------
```

Comment:
Nit: What's the purpose of the `//------` line? I know it's used all over the place in `System.Data`, but I'd get rid of it for new files (and eventually clean up all the other files at some point).